src/app/AppInitGate.test.tsx

```diff
@@ -96,4 +102,14 @@ describe('I18nGate', () => {
     expect(mockedI18n.initI18n).toHaveBeenCalledTimes(1)
     expect(navigateToError).toHaveBeenCalledWith('appInitFailed', 'some error')
   })
+
+  it('should show error screen in case of failed analytics init', async () => {
```

Comment:
this should never happen because we wrap both segment and statsig inits in try catches [here](https://github.com/valora-inc/wallet/blob/main/src/analytics/ValoraAnalytics.ts#L108-L152). We could also add another try / catch wrapper for ValoraAnalytics.init and don't throw error in case of failure and continue with app load. Thoughts?